distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -241,6 +244,31 @@ type File
     / : (Text | File) -> File
     / self subpath = self.resolve subpath
 
+    ## Join two or more path segments together.
```

Comment:
The current doc string does not mention normalize. I was surprised this is done when I noticed it in the tests. I think we should mention it someway.

```suggestion
    ## Join two or more path segments together, normalizing the `..` and `.` subpaths.
```